src/assets/docs/getting-started-with-cwl.md

```diff
@@ -305,14 +305,14 @@ So every time you do a commit to your GitHub repo Quay automatically builds and
 
 It will also ask you where your Dockerfile is located and where your build context is (normally the root). 
 
-At this point you can confirm your settings and "Create Trigger" followed by "Run Trigger Now" to actually perform the initial build of the Docker images.  You'll need to click on the little gear icon next to your build trigger to accomplish this.
+At this point, you can confirm your settings and "Create Trigger" followed by "Run Trigger Now" to actually perform the initial build of the Docker images.  You'll need to click on the little gear icon next to your build trigger to accomplish this.
 
 ![Manual Trigger](../assets/images/docs/manual_trigger.png)
 
-
+<!-- Improve the wording of this line -->
 Build it for `1.25-6_1.1` for this tutorial.  Typically, I build for each release and develop aka latest is built next time I check-in on that branch.
```

Comment:
Maybe "Manually trigger it with a version name of `1.25-6_1.1` for this tutorial. Normally, I let the build trigger build a new tag for each new release on GitHub. "latest" on Quay.io is built any time I check-in on any branch. This can be useful for development but is discouraged in favour of a tagged version number for formal releases of your tool"